pipelines/build/common/config_regeneration.groovy

```diff
@@ -291,6 +291,7 @@ class Regeneration implements Serializable {
                 ADOPT_BUILD_NUMBER: "",
                 ENABLE_TESTS: true,
                 ENABLE_INSTALLERS: true,
+                ENABLE_SIGNER = true,
```

Comment:
```suggestion
                ENABLE_SIGNER: true,
```

Reply:
Sorry for so many typos.
I added pr-tester in our environment now I am trying to figure out how to verify these locally,  next time I would verify locally to be more efficient.